frmts/zarr/zarr_v2_array.cpp

```diff
@@ -1155,9 +1155,11 @@ static GDALExtendedDataType ParseDtype(const CPLJSONObject &obj,
             }
             else if (chType == 'f' && nBytes == 2)
             {
+                // elt.nativeType = DtypeElt::NativeType::IEEEFP;
```

Comment:
exposing has Float32 will still be needed for environments without native Float16 support